test/refract/test-JsonSchema.cc

```diff
@@ -0,0 +1,706 @@
+//
+//  test/refract/test-JsonSchema.cc
+//  test-librefract
+//
+//  Created by Thomas Jandecka on 17/02/2018
+//  Copyright (c) 2018 Apiary Inc. All rights reserved.
+//
+
+#include "catch.hpp"
+
+#include "refract/JsonSchema.h"
+#include "refract/Element.h"
+#include "refract/JSONSchemaVisitor.h"
+#include "utils/so/JsonIo.h"
+#include "utils/log/Trivial.h"
+
+#include <chrono>
+#include <sstream>
+
+using namespace drafter::utils;
+using namespace drafter::utils::log;
+using namespace so;
+using namespace std::chrono;
+using namespace refract;
+
+namespace
+{
+    std::string to_string(const so::Value& v)
+    {
+        std::ostringstream ss{};
+        so::serialize_json(ss, v, so::packed{});
+        return ss.str();
+    }
+}
+
+SCENARIO("JSON Schema serialization of NullElement", "[json-schema]")
+{
+    GIVEN("An empty NullElement")
+    {
+        auto el = make_empty<NullElement>();
+
+        WHEN("a JSON Schema is generated from it")
+        {
+            auto result = schema::generateJsonSchema(*el);
+
+            THEN("the schema matches any boolean")
+            {
+                REQUIRE(to_string(result) == R"({"$schema":"http://json-schema.org/draft-04/schema#","type":"null"})");
+            }
+        }
+    }
+
+    GIVEN("A BooleanElement with content")
```

Comment:
BooleanElement? There is NullElement

Reply:
I have this fixed in tjanc/schema-definitions. Is it enough to fix it in the next PR?

Reply:
ok